src/overlays/actors/ovl_En_Fsn/z_en_fsn.c

```diff
@@ -1003,7 +1003,7 @@ void EnFsn_GiveItem(EnFsn* this, GlobalContext* globalCtx) {
 }
 
 void EnFsn_SetupResumeInteraction(EnFsn* this, GlobalContext* globalCtx) {
-    if (((void)0, gSaveContext.inventory.questItems) & gBitFlags[18]) {
+    if (CHECK_QUEST_ITEM(18)) {
```

Comment:
We should decide on hex vs dec for these, although it'll hopefully be redundant once we have a proper enum anyway.

Reply:
Redundant when we have an enum. Just wait for that to standardize. However, we should use the macro when it matches.